diesel/src/query_builder/debug.rs

```diff
@@ -2,7 +2,7 @@ use backend::Debug;
 use super::{QueryBuilder, BuildQueryResult};
 
 #[doc(hidden)]
-#[derive(Debug)]
+#[derive(Debug, Default)]
```

Comment:
Interesting. I agree, I would remove that `fn new() -> Self`.

I rarely have `fn new() -> Self` (i.e. with no parameter) but often use `fn new(stuff) -> Self` as a constructor (which may internally do stuff like `StructName { field1: stuff, ... StructName::default() }`).